### PR TITLE
Use proper GeoJSON types and fix display of Regions

### PR DESCRIPTION
**Pull request by banesullivan**
@annehaley - this fixes the issues displaying the polygons for Regions.

Note the new dependency for GeoJSON types. I still need to debug/fix something I broke with drawing polygons on the map

Note base branch

- [ ] Change region footprint to outline only
- [ ] Select a region - all sites appear as footprints

### CODE REVIEW

File: src/components/organisms/CesiumViewer.vue [hunk 1]
```diff
@@ -11,12 +11,13 @@ import {
 } from '@/store';
 import { cesiumViewer } from '@/store/cesium';
 import { Entity } from 'cesium';
+import { Position } from 'geojson';
```
Comment (annehaley) [inline src/components/organisms/CesiumViewer.vue:14]: I think this value is never used

**Review by annehaley — COMMENTED** [review]
This works to fix the mulitpolygon issue I was experiencing on the US entries. I really like the flyTo, as well.

I think the problem with resolving the `@types/geojson` package is due to us not having a `eslintrc.json` file in the repo. The name resolution was fixed when I added this, so I'll push it up here.

Still going through to find the problem with the `GeoJsonForm` component...

**Comment by annehaley** [comment]
Then again, introducing the `eslintrc.json` file makes new linting errors arise, now that it scrutinizes more files... Perhaps there is a good middle ground?

**Comment by banesullivan** [comment]
@annehaley, would you please take a go at this and resolve the merge conflict?

File: src/components/organisms/CesiumViewer.vue [hunk 2]
```diff
@@ -209,100 +206,13 @@ export default defineComponent({
...
-      }
+      Cesium.Camera.DEFAULT_VIEW_FACTOR = 0;
     });
 
     watch(specifiedShape, () => {
-      const uploadedFootprint: number[] = [];
-      specifiedShape.value.coordinates[0].forEach((e: number[]) => {
+      const uploadedFootprint: Cartesian3[] = [];
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      specifiedShape.value.coordinates.forEach((e: any) => {
```
Comment (zachmullen) [inline src/components/organisms/CesiumViewer.vue:215]: Seems like we can just do

```typescript
const uploadedFootprint: Cartesian3[] = specifiedShape.value.coordinates.map(
   ...
```

File: src/store/cesium.ts [hunk 3]
```diff
@@ -20,9 +23,11 @@ Cesium.RequestScheduler.maximumRequestsPerServer = 3;
 
 export const cesiumViewer = ref();
 
+export const useMap = ref(false);
+
 export const tileImageParams: Record<string, TileParamsType> = {};
 
-const footprintEntities: Record<string, Entity> = {}; // Cesium.Entity
+const footprintEntities: Record<string, Array<Entity>> = {}; // Cesium.Entity
```
Comment (zachmullen) [inline src/store/cesium.ts:30]: Is `Array<Entity>` equivalent to `Entity[]`? If so, let's just consistently use the latter notation.

File: src/store/cesium.ts [hunk 4]
```diff
@@ -99,19 +104,34 @@ watch(visibleOverlayIds, () => {
...
-  });
+export const addGeojson = (geojson: Polygon | MultiPolygon): Entity[] => {
+  const makeEntity = (
+    coordinates: Position[][],
+    parent: Entity | undefined = undefined,
+  ): Entity => {
+    const cesiumPoints: RGDResult[] = [];
+    coordinates[0].forEach((e: Position) => {
+      cesiumPoints.push(Cesium.Cartesian3.fromDegrees(e[0], e[1]));
+    });
```
Comment (zachmullen) [inline src/store/cesium.ts:115]: We can also convert this to use `map` to assign `cesiumPoints`

File: src/store/cesium.ts [hunk 5]
```diff
@@ -99,19 +104,34 @@ watch(visibleOverlayIds, () => {
...
+          Cesium.Color.fromRandom({ alpha: 0.5 }),
+        ),
+      },
+    });
+  };
+  if (geojson.type === 'MultiPolygon') {
+    const entities: Entity[] = [];
+    geojson.coordinates.forEach((a: Position[][]) => {
+      entities.push(makeEntity(a));
+    });
```
Comment (zachmullen) [inline src/store/cesium.ts:130]: Also convert to `map`

**Review by zachmullen — CHANGES_REQUESTED** [review]

File: src/components/organisms/CesiumViewer.vue (hunk 2 above)
Comment (annehaley) [reply src/components/organisms/CesiumViewer.vue:215]: Getting an error "this expression is not callable" on the map statement?

I attempted it like this:
```
const uploadedFootprint: Cartesian3[] = specifiedShape.value.coordinates.map(
        (e: any) => Cesium.Cartesian3.fromDegrees(e[0], e[1]),
);
```

Comment (zachmullen) [reply src/components/organisms/CesiumViewer.vue:215]: Is `coordinates` an Array? I assumed if it was valid to call `forEach` on it, then `map` would work too.

File: src/store/cesium.ts (hunk 5 above)
Comment (annehaley) [reply src/store/cesium.ts]: Changed in 73d01e4

File: src/store/cesium.ts (hunk 4 above)
Comment (annehaley) [reply src/store/cesium.ts]: Changed in 73d01e4

File: src/store/cesium.ts (hunk 3 above)
Comment (annehaley) [reply src/store/cesium.ts]: Changed in 73d01e4

**Comment by annehaley** [comment]
I fixed the merge conflicts, but searches still receive an error when any `q` is specified: "Invalid geometry value"

File: src/components/organisms/CesiumViewer.vue (hunk 2 above)
Comment (annehaley) [reply src/components/organisms/CesiumViewer.vue:215]: `coordinates` is typed as `number[][][] | number [][][][]`. I thought map would work on it as well, but it gives the following compile error: 

```
Each member of the union type '(<U>(callbackfn: (value: number[][], index: number, array: number[][][]) => U, thisArg?: any) => U[]) | (<U>(callbackfn: (value: number[][][], index: number, array: number[][][][]) => U, thisArg?: any) => U[])' has signatures, but none of those signatures are compatible with each other.
```

Comment (banesullivan) [reply src/components/organisms/CesiumViewer.vue]: I changed this to use `Cesium.GeoJsonDataSource` to create the `Entity` from our `Polygon` in e757ee6

**Pull request merged**